packages/ui-table/src/Table/README.md

```diff
@@ -1136,93 +1997,218 @@ Also you need the following props on the components:
 
 Custom table with `stacked` layout support:
 
-```javascript
----
-type: example
----
+- ```javascript
+  class CustomTableCell extends React.Component {
+    static contextType = TableContext
+
+    render() {
+      const isStacked = this.context.isStacked
+      if (isStacked) {
+        let headerTxt
+        if (typeof this.props.header === 'function') {
+          headerTxt = React.createElement(this.props.header)
+        } else {
+          headerTxt = this.props.header
+        }
+        return (
+          <div role="cell">
+            {headerTxt && headerTxt}
+            {headerTxt && ': '}
+            {this.props.children}
+          </div>
+        )
+      }
+      return <td>{this.props.children}</td>
+    }
+  }
+
+  class CustomTableRow extends React.Component {
+    static contextType = TableContext
+    state = { isHovered: false }
+
+    toggleHoverOff = () => {
+      this.setState({ isHovered: false })
+    }
+    toggleHoverOn = () => {
+      this.setState({ isHovered: true })
+    }
+
+    render() {
+      const { hover, headers, isStacked } = this.context
+      const Tag = isStacked ? 'div' : 'tr'
+      const rowStyle =
+        hover && this.state.isHovered
+          ? { backgroundColor: 'SeaGreen' }
+          : { backgroundColor: 'white' }
+
+      return (
+        <Tag
+          style={rowStyle}
+          role={isStacked ? 'row' : undefined}
```

Comment:
This role prop is missing from the functional example

Reply:
Added